components/challenges/header/Header.jsx

```diff
@@ -1,38 +1,105 @@
 import React from 'react';
 import Link from 'next/link';
-import Image from 'next/image';
 import Logop from '../../../public/icons/LogoPlataforma1.png';
-import styles from '../../../styles/Header.module.scss';
-import position from '../../../styles/Profile.module.scss';
 import { useUser } from '@auth0/nextjs-auth0'
+import styled from '@emotion/styled';
+import Image from 'next/image';
 
 
+const CotainerHeader = styled.div`
```

Comment:
LGTM, just one comment, seems that there is a little typo here in the **CotainerHeader** name, should be **ContainerHeader**, right?

Reply:
Yes there is a typo